build/rbac/get-helm-rbac.sh

```diff
@@ -27,6 +27,6 @@ done
 
 echo "generating Helm template with options: ${options[*]}" &>/dev/stderr
 
-${HELM} template ../../deploy/charts/rook-ceph "${options[@]}" | ./keep-rbac-yaml.sh
+${HELM} template ../../deploy/charts/rook-ceph "${options[@]}" --debug | ./keep-rbac-yaml.sh
```

Comment:
With this option, I see the additional logging output in the CI:
```
install.go:178: [debug] Original chart version: ""
install.go:195: [debug] CHART PATH: /Users/travisn/src/go/src/github.com/rook/rook/deploy/charts/rook-ceph
```
Do you expect other helpful info will be output in case of failure, or is this the info you need?

Reply:
i expect helpful debug info to be printed when yml parsing fails, for eg, on deliberately adding a failure in clusterrole.yml we get( 
```
/tmp/tmp.lzkyW0amE2 ~/rook/build/rbac
install.go:173: [debug] Original chart version: ""
install.go:190: [debug] CHART PATH: /home/deepika/rook/deploy/charts/rook-ceph

walk.go:74: found symbolic link in path: /home/deepika/rook/deploy/charts/rook-ceph/charts/library resolves to /home/deepika/rook/deploy/charts/library
Error: parse error at (rook-ceph/templates/clusterrole.yaml:1): unexpected "}" in operand
helm.go:81: [debug] parse error at (rook-ceph/templates/clusterrole.yaml:1): unexpected "}" in operand
.
Number of RBAC resources: 1
~/rook/build/rbac
make: *** [Makefile:195: gen-rbac] Error 1
```